pydoctor/utils.py

```diff
@@ -59,8 +63,11 @@ def resolve_path(path: str) -> Path:
     return Path(Path.cwd(), path).resolve()
 
 def parse_path(value: str, opt: str) -> Path:
-    """Parse a str path to a L{Path} object
+    """
+    Parse a str path to a L{Path} object
     using L{resolve_path()}.
+
+    Watch out, prints a message and SystemExits on error!
```

Comment:
Maybe ... but I think that error should be rename to `panic` :)

```diff
- raise error(f"{opt}: invalid path, {ex}.")
+ error(f"{opt}: invalid path, {ex}.")
```

Reply:
Not sure to understand your diff here, there is no "raise error" in the file.
I'm OK renaming error() to panic().

Reply:
there is at line 75 

https://github.com/twisted/pydoctor/pull/551/files#diff-ba8141603c14d5cbbbafb27976d7aa8cde126ac102147640ac1158c1f8d298c6R75